packages/shared/src/medical/webhook/webhook-request.ts

```diff
@@ -17,11 +17,17 @@ export type DocumentConversionWebhookType = z.infer<typeof docConversionWebhookT
 export const docBulkDownloadWebhookTypeSchema = z.literal(`medical.document-bulk-download-urls`);
 export type DocumentBulkDownloadWebhookType = z.infer<typeof docBulkDownloadWebhookTypeSchema>;
 
+export const hl7NotificationWebhookTypeSchema = z
+  .literal(`medical.hl7.admit`)
+  .or(z.literal(`medical.hl7.discharge`));
+export type Hl7WebhookTypeSchemaType = z.infer<typeof hl7NotificationWebhookTypeSchema>;
```

Comment:
nit;
export const hl7NotificationWebhookTypeSchema = z.enum([
  "medical.hl7.admit",
  "medical.hl7.discharge"
]);

zod infers it to the exact same type, but a bit more clear for these "many possible values for a string" cases